source/filter.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const map = {
+  "&": "&amp;",
+  "<": "&lt;",
+  ">": "&gt;",
+  '"': "&quot;",
+  "'": "&#39;",
+};
+
+
+/**
+ * Экранирует символы в html, оставляя только разрешенные теги
+ * @param {string} text - фильтруемый текст
+ * @param {Array<string>} tags - список разрешенных тегов
+ * @returns {string} отфильтрованный текст
+ */
+const filter = (text, tags = []) => {
+  if (!(typeof text === "string")) {
+    throw new Error("expected String");
```

Comment:
```suggestion
    throw new TypeError("expected String");
```
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/TypeError